### DISCUSSION TITLE
Functioning Hardware ***  Please vote!!!

### DISCUSSION THREAD

**Discussion by tom76017** [GerryDazoo/Slinger#404] [Polls]
I'm curious about how many users have had success in setting up a Slingbox with Gerry's solution.  While there seems to be some issues with setting up both boxes that went offline at N9 and new purchases from eBay, I think that there has been a huge success rate also.  My M1 is great, but I don't use the remote control option.

**_This poll is not for discussions.  I suggest that you either open a new discussion or add to an existing GENERAL discussion._**

**Comment by GerryDazoo** [comment]
You can only select one! And only once. I've got 3 boxes I use regularly.

**Reply by tom76017** [reply]
Yes, I think it's a limitation of the polling process git implemented.  Just radio buttons.  Most young coders don't even know what a radio button is.  It's just an widget.   They should have used check boxes.  I was able to select _Hide Results_ at the bottom to change my selection.

**Reply by Toei79** [reply]
i have two 350 and one sb300. working great both , im no do the block port thing yet. 
sb300 i dont check yet , im guessing this one needs block ports. 

also i get one 500 from ebay.

**Reply by tom76017** [reply]
**_This poll is not for discussions.  I suggest that you either open a new discussion or add to an existing GENERAL discussion._**

**Comment by pfuller88** [comment]
1 500, 1 proHD, 1 pro working.  I have an M1 and solo bricked.

**Reply by GerryDazoo** [reply]
As far as I know, the 500's 350's and M1 were not bricked by Sling.
My M1 is still up and running. There may be something else wrong with your M1.

**Reply by pfuller88** [reply]
i do not have the password to the M1 and for the life of me I cannot reset the box.  i’ve tried about 15 times, holding the reset button for a long time, short time, etc..  each time it comes back and will not accept the default password or with an error.  i unplug the box to restart and the old user comes back.

**Reply by tom76017** [reply]
**_This poll is not for discussions.  I suggest that you either open a new discussion or add to an existing GENERAL discussion._**

**Comment by BobC1954** [comment]
I have 2 ProHD (one was isolated so never bricked), other unbricked. And 1 Echostar HDS-600rs also isolated and never bricked. All working fine currently.

**Reply by tom76017** [reply]
**_This poll is not for discussions.  I suggest that you either open a new discussion or add to an existing GENERAL discussion._**

**Comment by floormanager1** [comment]
Unbricked: ProHD, M1 & 350
Bricked: Solo and Pro

**Reply by tom76017** [reply]
**_This poll is not for discussions.  I suggest that you either open a new discussion or add to an existing GENERAL discussion._**

**Comment by sp222222** [comment]
the answer to your poll is all of the above ! ! ! 

I've tested the 240, 500, ProHD and Solo and all work fine with slinGER. 

You won't get a straight button answer b/c too many of us have bricked some of these things. (and no, M1 is not brickable at this time).  I have bought 3 solos off of ebay post N9 and as long as blocking rules are in place for the IP the slingbox uses, slinGER works on all those solos. I also, followed the pdf proudX made and was able to UNbrick a ProHD and scan for channels and save said HD channels

**Comment by remgr** [comment]
As others have said, I have more than one slingbox.  I have any M1, and M2 and a 350 all working perfectly.  I bought a ProHD from ebay as insurance.  It is the only one that can be successfully unbricked.  It is stored for now.

Once again thank you Gerry!
